packages/front-end/components/Experiment/MetricsSelector.tsx

```diff
@@ -247,7 +247,14 @@ const MetricsSelector: FC<{
         try {
           const clipboard = e.clipboardData;
           const data = JSON.parse(clipboard.getData("Text"));
-          if (data.every((d) => d.startsWith("met_") || d.startsWith("mg_"))) {
+          if (
+            data.every(
```

Comment:
Two questions I have (but not necessarily for this PR) are:
- Should this be `every` or should we also succeed partially if we can?
- Would we benefit of having these significant prefixes list somewhere and consume it here so this doesn't happen again?

Reply:
I could go either way on every vs some.  The only real magic prefix is `fact__` (and we do have a helper method already that checks if something is a fact metric id). The rest are just conventions and not strictly enforced. So there are some metrics that don't start with `met_` for example.